autoload/neomake/makers/ft/slim.vim

```diff
@@ -0,0 +1,13 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#slim#EnabledMakers()
+    return ['slimlint']
+endfunction
+
+function! neomake#makers#ft#slim#slimlint()
+    return {
+        \ 'exe': 'slim-lint',
+        \ 'args': ['--no-color'],
+        \ 'errorformat': '%f:%l %m'
```

Comment:
There will be no error type with this.

You could use `%t%m` instead of `%m` - depending on the format.

Can you provide some example output?
Are there different types after all?